docs/config/README.md

```diff
@@ -25,6 +25,15 @@ $ tree ~/.minio
 ```
```

Comment:
Need to remove `config.json` from here in `tree` output

Reply:
done.